elements/pfe-sass/mixins/_copy-mixins.scss

```diff
@@ -174,20 +183,46 @@
     $sizing: "#{$sizing}";
 
     // Adds !important to the end of the property; useful for utility classes only. Use with caution.
+    $local-region: null;
```

Comment:
You don't need to remap the region value, you can pass that in directly.  It was good as it was - I'm wondering why you updated this?